server/speakerList.json

```diff
@@ -1 +1 @@
-{"speakers":["bristech", "danfairs","michaellnorth","cloud8421","pimterry","samhogy","AlexEvade","FredAtBootstrap","shaundunne","SteffLocke","tojarrett","daveellender","sublimino","jezenthomas","SimonMoffatt"]}
+{"speakers":["bristech","danfairs","michaellnorth","cloud8421","pimterry","samhogy","AlexEvade","FredAtBootstrap","shaundunne","SteffLocke","tojarrett","daveellender","sublimino","jezenthomas","SimonMoffatt","d3v_rand0m"]}
```

Comment:
Not that I know of, although it wouldn't really hurt to just .gitignore the json files. Think it's a good idea?